demisto_sdk/commands/common/hook_validations/docker.py

```diff
@@ -337,3 +342,69 @@ def parse_docker_image(self, docker_image):
                 return 'demisto/python', self.get_docker_image_latest_tag('demisto/python', None)
             else:
                 return 'demisto/python3', self.get_docker_image_latest_tag('demisto/python3', None)
+
+    @staticmethod
+    def get_docker_image_latest_tag_from_iron_bank_request(docker_image_name):
+        """
+        Get the latest tag for a docker image by request to Iron Bank Repo.
+        Args:
+            docker_image_name: The docker image name.
+
+        Returns:
+            The latest tag for the docker image.
+        """
+        project_name = docker_image_name.replace('demisto/', '')
+        api_url = 'https://repo1.dso.mil/api/v4/projects/dsop%2Fopensource%2Fpalo-alto-networks%2Fdemisto%2F'
+        commits_url = api_url + f'{project_name}/pipelines'
+        manifest_url = api_url + f'{project_name}/repository/files/hardening_manifest.yaml/raw'
+
+        try:
+            last_commit = DockerImageValidator._get_latest_commit(commits_url, docker_image_name)
+            if not last_commit:
+                return ''
+            manifest_file_content = DockerImageValidator._get_manifest_from_commit(manifest_url, last_commit)
+            if not manifest_file_content:
+                return ''
+        except Exception as e:
+            raise(e)
```

Comment:
This has no much impact, better to use logging for debugging purposes.

Reply:
we cannot actually add logging (correct me if i'm wrong please) except for handle_error, which we cannot use because of the format command using the same functions. The Exception allows us to see the reason for fail in both format and validate command.

Reply:
So what should the user see in a case of an exception?

Reply:
The Format will show the error I am raising.  The validate will show it cannot fetch the tag (a more generic error, same as the regular docker hub fetching error). I can edit the current generic error to include the exception I raised if you think this is the best solution.

Reply:
I believe it's better to let the user know why the validation is failing in that case and what should he do in order to solve it.